Dockerfile

```diff
@@ -1,13 +1,13 @@
-# Docker image for the Drone build runner
-#
-#     CGO_ENABLED=0 go build -a -tags netgo
-#     docker build --rm=true -t plugins/drone-cache .
+FROM centurylink/ca-certs
```

Comment:
Also stop using this image?

Reply:
why stop using this image?

Reply:
Because it's not maintained for more than a year, drone will also use a different image.

Reply:
I see you build the image using docker multiple stages mode.